src/components/accounts/SetupSeedPhraseForm.js

```diff
@@ -67,7 +68,7 @@ const SetupSeedPhraseForm = ({
                     <IconMCopy color='#6ad1e3' />
                 </FormButton>
                 <FormButton
-                    linkTo={`/setup-seed-phrase/${accountId}/verify/${fundingContract ? `${fundingContract}/${fundingKey}/` : ``}`}
+                    linkTo={linkTo ? linkTo : `/setup-seed-phrase/${accountId}/verify/${fundingContract ? `${fundingContract}/${fundingKey}/` : ``}`}
```

Comment:
nit: should we always pass `linkTo` as parameter (maybe as `continueLinkTo`)? Looks like this component itself doesn't really use `accountId` / `fundingContract` / `fundingKey` otherwise.

Reply:
Yes, for the implicit account some components are overkill, but this was the easiest working model of seed phrase.

Reply:
@mattlockyer not sure what do you mean. I'm suggesting to have this component always take `linkTo` option (renamed as `continueLinkTo`). Then this whole thing can be always passed from outside:

```
`/setup-seed-phrase/${accountId}/verify/${fundingContract ? `${fundingContract}/${fundingKey}/` : ``
```

Feels kinda weird to pass it from outside sometimes and not pass in other times.

Reply:
It's used by a completely different flow, setupSeedPhrase.

I'm just borrowing the component, I'm not fixing these other components as part of this PR.